README.md

```diff
@@ -280,12 +283,12 @@ webpack:
 webpack:
     output:
         path: '%kernel.root_dir%/../web'
-        dump_path: '%kernel.root_dir%/../web/bundles'
+        dump_path: '/bundles'            # public assets will be copied to '%kernel.root_dir%/../web/bundles'
```

Comment:
Then you would configure `path: '%kernel.root_dir%/../httpdocs'` and would make your resolved dump_path `path: '%kernel.root_dir%/../httpdocs/bundles'`